slimish_jinja/slimish_jinja.py

```diff
@@ -1,5 +1,11 @@
 import os.path
-from StringIO import StringIO
+
+from uncompyle6 import PYTHON3
```

Comment:
you dont need add a dependency on `uncompyle6` to determine `PYTHON3`.

Use `sys.version_info`